index.js

```diff
@@ -22,6 +22,50 @@ const removeElements = elements => {
 	}
 };
 
+const scrollToElement = (element, options) => {
+	const isOverflown = e => e.scrollHeight > e.clientHeight || e.scrollWidth > e.clientWidth;
+	const findScrollParent = element => {
+		if (element === undefined) {
+			return;
+		}
+
+		if (isOverflown(element)) {
+			return element;
+		}
+
+		return findScrollParent(element.parentElement);
+	};
+
+	const calculateOffset = (rect, options) => {
+		if (options === undefined) {
+			return {x: rect.left, y: rect.top};
+		}
+
+		const offset = options.offset | 0;
+
+		switch (options.offsetFrom) {
+			case 'top':
+				return {x: rect.left, y: rect.top + offset};
+			case 'right':
+				return {x: rect.left - offset, y: rect.top};
+			case 'bottom':
+				return {x: rect.left, y: rect.top - offset};
+			case 'left':
+				return {x: rect.left + offset, y: rect.top};
+			default:
+				return {x: rect.left, y: rect.top};
```

Comment:
I think it would be better to throw here. If an object is given, the user should specify all the properties.